### ISSUE TITLE
Unable to select a form in a Formie field without managing rights to the form

### ISSUE THREAD

**Issue by thijskaspers** [verbb/formie#484]
A client of ours can't select a form from a Formie field that we've added to several entries. We want the client to have the ability to select a form to place it on a page, but the popup that should show the forms is empty.

We've found out that you can only see a form when you have managing rights to the form, but we don't want the client to have full rights to edit the forms, that is too technical for them. Being able to manage form settings, or being able to select it to include it on a page should be 2 different things right?

**Steps to reproduce**
1. Create a form
2. Add a form field to an entry to be able to select forms
3. Create a new user with access to Formie and Subsmission for example, but without managing rights to edit the forms
4. Let that user select a form at the entry through the created form field. It's blank.

**Additional info**
- Plugin version: 1.4.5
- Craft version: 3.6.16

## Additional question ##
Something else, why is it that you can't select Forms at the Formie field settings? It shows the following 'sources' with only an All option:

![Schermafbeelding 2021-06-07 om 12 28 53](https://user-images.githubusercontent.com/1608184/121001840-fcc3bd00-c78b-11eb-9dfa-9a17b4c66de6.png)

It would be nice that the forms are the sources, so you can set up which Forms the clients can choose from? You don't want them to be able to insert any form on any page, not all are compatible. Or are we missing something here?

Thanks!

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.5",
  "...": "..."
}
```

Then run `composer update`.

As for the sources, this follows the pattern of Craft elements, where a source is a collection of elements. Similar to a section being a collection of entries, volumes a collection of assets, category groups a collection of categories, etc. As such, for Formie, forms are at the moment grouped by their Form Template. I'll look at adding singular forms to pick from in the future.

**Comment by thijskaspers** [comment]
Thanks Josh, that was quick! We've pulled dev-craft-3 and it works :)

About the sources, single forms would be awesome so we can restrict the client in what forms they can pick, but it's only a 'nice to have'. We fixed it for now by checking the formhandle in the template and if it's unallowed it won't render.

**Comment by engram-design** [comment]
No problem, keeping open for now until the formal release.

**Comment by engram-design** [comment]
Fixed in [1.4.6](https://github.com/verbb/formie/releases/tag/1.4.6)
